balrogscript/src/balrogscript/submitter/cli.py

```diff
@@ -143,24 +148,50 @@ def generate_data(self, appVersion, productName, version, buildNumber, updateCha
     def run(
         self, appVersion, productName, version, buildNumber, updateChannels, ftpServer, bouncerServer, enUSPlatforms, hashFunction, updateLine, **updateKwargs
     ):
-        data = self.generate_data(
+        blob = self.generate_data(
             appVersion, productName, version, buildNumber, updateChannels, ftpServer, bouncerServer, enUSPlatforms, updateLine, **updateKwargs
         )
         name = get_release_blob_name(productName, version, buildNumber, self.suffix)
-        api = Release(name=name, auth0_secrets=self.auth0_secrets, api_root=self.api_root)
-        try:
-            current_data, data_version = api.get_data()
-        except HTTPError as e:
-            if e.response.status_code == 404:
-                log.warning("Release blob doesn't exist, using empty data...")
-                current_data, data_version = {}, None
-            else:
+
+        if self.backend_version == 2:
+            log.info("Using backend version 2...")
+            blob["schema_version"] = self.schemaVersion
+            blob["hashFunction"] = hashFunction
+            blob["name"] = name
+            data = {"product": productName, "blob": blob}
+            headers = {"Accept-Encoding": "application/json", "Accept": "application/json", "Content-Type": "application/json", "Referer": self.api_root}
```

Comment:
Are you sure you need to set these? In my curl tests, only `Content-Type` is needed, and if you pass a dictionary to `requests_api.put`, it should set that for you, eg: `requests_api.put(url, json=data)`.

Reply:
I wasn't sure these were needed, no.